ui/src/components/item/QList.test.js

```diff
@@ -0,0 +1,171 @@
+import { mount, flushPromises } from '@vue/test-utils'
+import { describe, test, expect } from 'vitest'
+
+import QList from './QList.js'
+
+describe('[QList API]', () => {
+  describe('[Props]', () => {
+    describe('[default attributes]', () => {
+      test('has a role="list" attribute with a div', async () => {
+        const wrapper = mount(QList)
+
+        const target = wrapper.get('.q-list')
+
+        expect(target.element.getAttribute('role')).toBeDefined()
+        expect(target.element.getAttribute('role')).toBe('list')
```

Comment:
Being equal to `list` implies being defined
```suggestion
        expect(target.element.getAttribute('role')).toBe('list')
```